compute/admin_guide/vulnerability_management/registry_scanning/nexus-registry.adoc

```diff
@@ -0,0 +1,116 @@
+== Sonatype Nexus Registry
+
+Scan Nexus registries.
+
+To scan a repository in Sonatype Nexus Registry, create a new registry scan setting.
+
+[#nexus-repo-connector]
+==== Prerequisites
+
+* You have xref:../../install/defender_types.adoc#[installed a Defender] somewhere in your environment. 
+
+* The Defender can establish a connection with the Nexus registry over port 443.
```

Comment:
Should this say the "Defender connects to the Nexus registry on port 443", or is this truly it can connect on 443 or any other port?

Reply:
The Defender can establish a connection with the Nexus registry over the connector port that you configured in the Nexus registry.